### PR TITLE
WT-12976 Set WT_SESSION_PREFETCH_THREAD for prefetch sessions

### PR DESCRIPTION
**Pull request by etienneptl**

### CODE REVIEW

**Review by mm-ng — APPROVED** [review]
LGTM

File: src/conn/conn_prefetch.c [hunk 1]
```diff
@@ -77,6 +77,9 @@ __wt_prefetch_thread_run(WT_SESSION_IMPL *session, WT_THREAD *thread)
     WT_ASSERT(session, session->id != 0);
     conn = S2C(session);
 
+    /* Mark the session as an eviction thread session. */
```
Comment (kommiharibabu) [inline src/conn/conn_prefetch.c:80]: The comment needs to be adjusted to `prefetch`.

Comment (etienneptl) [reply src/conn/conn_prefetch.c]: Yes sorry, just re-pushed.

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Pull request merged**